php/sunrise-router-annotations/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "require": {
+        "sunrise/http-router": "^2.4",
```

Comment:
could you use  `~2.4.0`, to ensure in **2.4**

Reply:
mission complete